collections/guides/quickstart_ruby/index.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: default
+title: Quickstart with Ruby
+permalink: /getting_started/quickstart_with_ruby/
+categories: quickstart
+nav_order: 1
+has_children: true
+has_toc: false
+description: Get Started with Basis Theory with Ruby
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+# Quickstart with Ruby
+
+In this tutorial, you will create a new Ruby script and use Basis Theory’s platform to secure a string of data and then read that data back out. Once you’ve completed this tutorial, you’ll have a foundational understanding of how to secure your data with Basis Theory allowing you to no longer worry about the underlying storage or encryption of the data.
+
+If you are already familiar with Basis Theory and our platform in general, our guide on [How to send Token data via HTTP](https://guides.basistheory.com/guides/use-token-data-in-http-requests/) or [Collecting payments with Elements](https://guides.basistheory.com/guides/collect-atomic-cards-with-elements/) may be a better place to start.
+
+## Step 1: Get your Ruby environment ready
+
+### Step 1.1: Ensure you have Ruby installed
+
+Completing this tutorial will require you to have Ruby installed on your local system or have an environment you can run your script in. If you’re looking for help getting Ruby installed for your system — checkout their guide on [installing](https://www.ruby-lang.org/en/documentation/installation/).
+
+### Step 1.2: Create a Ruby script
+To start, you’ll need a new Ruby file. Use the following commands (or however you create files) to create a file called `basistheory.rb`:
+
+```bash
+//Mac or Linux
+touch basistheory.rb
+
+//Windows
+type nul > basistheory.rb
+```
+
+### Step 2: Import needed dependencies
+
+The simplest form of securing data with Basis Theory is through our API with a JSON body. In Ruby, this requires a few different libraries to make an API call. At the top of your file, you’ll import the following dependencies:
+
+```ruby
+require 'uri'
+require 'net/http'
+require 'json'
+```
+
+## Step 3: Create a new Application in your Basis Theory Account
+
+If you don’t already have a Basis Theory account and your first Tenant, create one [here](https://portal.basistheory.com/register).
+
+Within your Basis Theory account, create a new Application. To create a new application, head to our portal [here](https://portal.basistheory.com/applications/create) — this Application’s API Key enables you to authenticate with the Basis Theory platform and create Tokens within your Tenant.
```

Comment:
```suggestion
Within your Basis Theory account, create a new Application. To create a new application, head to our portal [here](https://portal.basistheory.com/applications/create) — this Application’s API Key will enable you to authenticate with the Basis Theory platform and create Tokens within your Tenant.
```
future tense might be better here since we haven't created the application yet